src/main/java/org/dataloader/CacheMap.java

```diff
@@ -70,7 +72,7 @@
      *
      * @return the cached value, or {@code null} if not found (depends on cache implementation)
      */
-    V get(U key);
+    CompletableFuture<V> get(K key);
```

Comment:
The small brekaing change.

It was always a CompleteableFuture at runtime but now the signature reflects this more correctly